src/pages/learn/javascript/codemods-with-babel-plugins/index.mdx

```diff
@@ -1,6 +1,15 @@
 export const meta = {
   title: `Codemods with Babel Plugins`,
   contributors: ['Laurie Barth'],
+  coverImage: {
+    url:
+      'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607523756/egghead-next-ebombs/article-illustrations/codemods.png',
+    alt:
+      'abstract illustration of sphere morphing into ring, stop motion style',
+  },
```

Comment:
didnt forget to compress the image before uploading to cloudinary (💙 [alfred tinypng workflow](https://www.alfredapp.com/blog/productivity/featured-workflow-compress-images-with-tinypng/))